### PR TITLE
Upload notes and photos

### PR DESCRIPTION
**Pull request by rluck0419**
upload for photos, and not for notes - the story name was incorrect (photos only)

### CODE REVIEW

File: app/views/photos/index.html.erb [hunk 1]
```diff
@@ -2,8 +2,9 @@
   <h3>
     Photo <%= i + 1 %>
   </h3>
-  <div>Name: <%= photo.photo_with_type %></div>
-  <div>Author: <%= note.user_id %></div>
-  <div>Category: <%= note.category.name %></div>
-  <div>Body: <%= note.body %></div>
+  <div>Name: <%= photo.name %></div>
+  <div>Author: <%= photo.user_id %></div>
```
Comment (kate-walters) [inline app/views/photos/index.html.erb]: photo.user.name

File: app/views/photos/show.html.erb [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <h3>Note:</h3>
-<div>Name: <%= photo.photo_with_type %></div>
+<div>Name: <%= photo.name %></div>
 <div>Author: <%= photo.user_id %></div>
```
Comment (kate-walters) [inline app/views/photos/show.html.erb]: photo.user.name

**Comment by MattRice12** [comment]
+1

**Comment by kate-walters** [comment]
+1

**Pull request merged**